p2p/libp2p/ports.go

```diff
@@ -0,0 +1,89 @@
+package libp2p
+
+import (
+	"fmt"
+	"net"
+	"strconv"
+	"strings"
+
+	"github.com/ElrondNetwork/elrond-go/core/random"
+	"github.com/ElrondNetwork/elrond-go/p2p"
+)
+
+type testPortHandler = func(int) error
```

Comment:
this type def is not necessary. if yes rename.

Reply:
removed type definition. However, I kept the function pointer in order to better test the functionality.